peerlist-server/generate-config.py

```diff
@@ -40,6 +412,37 @@ def get_node_uuid(node_id, working_directory):
 
 
 def make_peerlist_entry(uuid, node_id, same_port=False):
+    
+    ####Add node to BluzelleDockerSwarm
+    nonce = w3.eth.getTransactionCount(acct.address)
+    node_name = "node_{}".format(node_id)
+    node_host = ([l for l in ([ip for ip in socket.gethostbyname_ex(socket.gethostname())[2] 
```

Comment:
Move this to function

Reply:
Give some thought to failure cases and where we see console errors within AWS

Reply:
Moved.  Changes reflected with the new commit.

For viewing errors on console, it's a configuration through the cloudformation script to include CloudWatch.